src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java

```diff
@@ -170,4 +138,43 @@
 		}
 		return testClassesAsArray;
 	}
+	
+	private void insertIntoTestClassesArray(String strLine, ArrayList<String> testClassList){
```

Comment:
A sentence or two on overview of the new method would help in maintenance effort